x-pack/plugins/canvas/init.js

```diff
@@ -45,17 +45,17 @@ export default async function(server /*options*/) {
         catalogue: ['canvas'],
         app: ['canvas', 'kibana'],
         savedObject: {
-          all: ['canvas'],
+          all: ['canvas-workpad'],
```

Comment:
Admittedly, I'm not super familiar with how this config works. Does this use tags or something on saved objects? And will this cause an issue with existing workpads?

Reply:
This restricts access to specific "saved object types", so if we were to grant the "canvas all" privilege, they would get all saved object related actions available for whatever types we specify in this array.

Reply:
@w33ble no impact for existing workpads. We don't use tags on the saved objects themselves here, so this configuration is transparent to the actual documents we index.